galsim/chromatic.py

```diff
@@ -1226,6 +1226,60 @@ def __init__(self, original, waves, oversample_fac=1.0, use_exact_sed=True,
 
         # Don't interpolate an interpolation.  Go back to the original.
         self.deinterpolated = original.deinterpolated
+    
+    @classmethod
+    def from_images(cls, images, waves, _force_stepk = None, _force_maxk = None, **kwargs):
+        """
+        Alternative initiazliation to InterpolatedChromaticObject from input images at specific wavelenghts. Any parameters accepted by
+        the InterpolatedImage class can be passed in kwargs. Note that stepk and maxk are parameters that can depend on the image size,
+        and therefore on the wavelength. If not given, as a list for every image, or a single number for all images, it will be caluclated
```

Comment:
```suggestion
        and therefore on the wavelength. If not given, as a list for every image, or a single number for all images, it will be calculated
```